ansible/roles/k8s_resources/tasks/istio.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Add the official istio helm repository
+  shell: "helm repo add istio.io {{ istio_chart_rep_url }}"
```

Comment:
--kube-context {{ cluster_name }} is missed

Reply:
Fixed